x-pack/plugins/security_solution/public/flyout/right/components/host_entity_overview.tsx

```diff
@@ -37,8 +37,14 @@ import {
 const StyledEuiBetaBadge = styled(EuiBetaBadge)`
   margin-left: ${({ theme }) => theme.eui.euiSizeXS};
 `;
+const HOST_ICON = 'storage';
 const CONTEXT_ID = `flyout-host-entity-overview`;
 
+const StyledEuiLink = styled(EuiLink)`
```

Comment:
can this be imported from expandable panel?

Reply:
same as other one, using emotion directly instead